sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -223,6 +223,7 @@ public partial class RequestFailedException : System.Exception, System.Runtime.S
         public RequestFailedException(string message) { }
         public RequestFailedException(string message, System.Exception? innerException) { }
         public string? ErrorCode { get { throw null; } }
+        public Azure.Response? Response { get { throw null; } }
```

Comment:
Would it make sense to use `GetRawResponse()` instead?  Looking at the [Core API](https://apiview.dev/Assemblies/Review/77bdbcef49a3465fa96fac3fb745bf57), `GetRawResponse()` is used on `Response<T>`, `Operation<T>` and `Pageable<T>` to access the raw Response.  `Response` as a property looks like it only appears on `HttpMessage`.  `RawResponse` doesn't show up anywhere, so I don't know if it would make sense to introduce that as a convention.  If it were a method instead of a property, it might also indicate that further processing like buffering is needed.

@JoshLove-msft, what motivated your preference for `Response` as a property over `GetRawResponse()`?

Reply:
If we do decide to make this `GetRawResponse` it may make more sense to call `BufferResponseIfNeeded()` just in time as mentioned in [this comment](https://github.com/Azure/azure-sdk-for-net/pull/35716#discussion_r1173910590)

Reply:
Yeah, that would be a nice side-effect if it makes sense -- I'm interested to learn what would change about the rest of the implementation if we had to wait on that, for example, the ability to parse the error message might be impacted.

Reply:
> If we do decide to make this `GetRawResponse()` it may make more sense to call `BufferResponseIfNeeded()` just in time as mentioned in [this comment](https://github.com/Azure/azure-sdk-for-net/pull/35716#discussion_r1173910590)

I don't think so because we would then only be disposing the network stream if the user calls GetRawResponse.

Reply:
I don't really have a preference of Response vs GetRawResponse().

Reply:
I would probably have a slight preference for GetRawReponse(), just from the perspective of matching what's in the more general usage API group, and potentially clarifying any possible confusion around whether it's a raw response vs. a Response\<T\>.

Reply:
Updated to use a method